Description: Search libs in private path
 Look for libraries first in LD_LIBRARY_PATH (as suggested
 by Michael Gold in http://bugs.debian.org/737365
 then in higans private lib path, then in the standard paths.
Author: Tobias Hansen <thansen@debian.org>

Index: higan-098/nall/dl.hpp
===================================================================
--- higan-098.orig/nall/dl.hpp	2016-05-26 17:26:08.052709716 +0200
+++ higan-098/nall/dl.hpp	2016-05-26 17:28:44.292107709 +0200
@@ -38,6 +38,7 @@
 inline auto library::open(const string& name, const string& path) -> bool {
   if(handle) close();
   if(path) handle = (uintptr_t)dlopen(string(path, "lib", name, ".so"), RTLD_LAZY);
+  if(!handle) handle = (uintptr_t)dlopen(string("/usr/lib/higan/lib", name, ".so"), RTLD_LAZY);
   if(!handle) handle = (uintptr_t)dlopen(string(userpath(), ".local/lib/lib", name, ".so"), RTLD_LAZY);
   if(!handle) handle = (uintptr_t)dlopen(string("/usr/local/lib/lib", name, ".so"), RTLD_LAZY);
   if(!handle) handle = (uintptr_t)dlopen(string("lib", name, ".so"), RTLD_LAZY);
